### PR TITLE
initial portfolio creation

### PR DESCRIPTION
**Pull request by dave5801**
I took a little extra time with this assignment largely to review conceptual information regarding git. I looked into what a .gitignore file was. I reviewed the purpose of pull requests. And purposefully redid the repository creation to make sure I understood everything.

The basic conceptual information is highly challenging.

### CODE REVIEW

**Comment by musflood** [comment]
You can build a .gitignore from scratch by using [gitignore.io](https://www.gitignore.io/). Usually adding the files for Mac, Linux, Windows, and Node are a good starting point.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+<body>
+
+<div class="topnav" id="myTopnav">
+  <a href="#home">About</a>
+  <a href="#news">Writings</a>
+  <a href="#contact">Projects</a>
+  <a href="javascript:void(0);" style="font-size:15px;" class="icon" onclick="toggleResp()">&#9776;</a>
+</div>
+
+<div style="padding-left:16px">
```
Comment (musflood) [inline index.html]: Please refrain from using inline styling.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
```
Comment (musflood) [inline index.html]: Be sure to add a viewport `meta` tag to your page.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+.topnav .icon {
+  display: none;
+}
+
+@media screen and (max-width: 600px) {
```
Comment (musflood) [inline index.html]: This adds styling for the mobile view, and its therefore not mobile-first styling. You should be using `min-width` and adding styling for the desktop instead.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+  .topnav a:not(:first-child) {display: none;}
+  .topnav a.icon {
+    float: right;
+    display: block;
+  }
+}
+
+@media screen and (max-width: 600px) {
```
Comment (musflood) [inline index.html]: You only need one `@media` query since both these queries have the same breakpoint.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+}
+</style>
+</head>
+<body>
+
+<div class="topnav" id="myTopnav">
+  <a href="#home">About</a>
+  <a href="#news">Writings</a>
+  <a href="#contact">Projects</a>
+  <a href="javascript:void(0);" style="font-size:15px;" class="icon" onclick="toggleResp()">&#9776;</a>
```
Comment (musflood) [inline index.html]: You should not be setting event listeners in the HTML. Those should be applied in the JS file.

**Comment by dave5801** [comment]
Made some changes, social media links, separate CSS and JS files.

**Pull request merged**